scroll-animations-1/Overview.bs

```diff
@@ -736,67 +736,79 @@ spec: cssom-view-1; type: dfn;
 
 	A set of animation keyframes can be attached
 	in reference to [=named timeline ranges=]
-	by adjusting the duration of the animation.
+	by restricting the to the specified timeline range.
+	Delays ('animation-start-delay'/'animation-end-delay')
+	are set within this restricted range.
+	If a duration ('animation-duration') is given,
+	it measures from the range-restricted start,
+	and further restricts the range-restricted end.
 
 	<pre class="propdef shorthand">
-		Name: animation-delay, animation-range
-		Value: [ <<'animation-delay-start'>> <<'animation-delay-end'>>? | <<timeline-range-name>> ]#
+		Name: animation-range
+		Value: [ <<'animation-range-start'>> <<'animation-range-end'>>? | <<timeline-range-name>> ]#
 	</pre>
 
-	The 'animation-delay' and 'animation-range' properties
-	are both [=shorthand property|shorthands=]
-	that set 'animation-delay-start' and 'animation-delay-end'
+	The 'animation-range' property is a [=shorthand property|shorthand=]
+	that sets 'animation-range-start' and 'animation-range-end'
 	together in a single declaration.
-	If the <<'animation-delay-end'>> value is omitted,
+	If the <<'animation-range-end'>> value is omitted,
 	it is set to zero.
 	If a <<timeline-range-name>> alone is specified,
-	'animation-delay-start' is set to that name plus ''0%''
-	and 'animation-delay-end' is set to that name plus ''100%''.
+	'animation-range-start' is set to that name plus ''0%''
+	and 'animation-range-end' is set to that name plus ''100%''.
 
 	<pre class="propdef">
-		Name: animation-delay-start
-		Value: [ <<time>> | <<timeline-range-name>> <<percentage>> ]#
-		Initial: 0s
+		Name: animation-range-start
+		Value: [ normal | <<timeline-range-name>> <<percentage>> ]#
+		Initial: normal
```

Comment:
I think maybe you meant auto here to align with the auto <dfn> below?

Reply:
I meant normal and failed to make it match below. :/ Fixed.